docs/reference/sso.md

```diff
@@ -0,0 +1,522 @@
+
+### AD FS (using SAML)
+{: .d-inline-block }
+Enterprise
+{: .label .label-purple }
+
+> **Note**: AD FS integration is using certificates to sign & encrypt requests going out from Fluffy and decrypt incoming requests from AD FS server.
+
+In order for fluffy to work, some values must be configured, update the relevant values section in this file and comment it out.
+1. Replace `fluffy.saml_rsa_public_cert` and `fluffy.saml_rsa_private_key` with real certificate values
+2. Replace `fluffyConfig.auth.saml.idp_metadata_url` to the metadata URL of the provider <adfs-auth.company.com>
+3. Replace `fluffyConfig.auth.saml.external_user_id_claim_name` with the claim name representing user id name in ADFS
+4. Replace `lakefs.company.com` with the lakeFS server endpoint.
+
+If you'd like to generate the certificates using OpenSSL, you can take a look at the following example:
+```sh
+openssl req -x509 -newkey rsa:2048 -keyout myservice.key -out myservice.cert -days 365 -nodes -subj "/CN=lakefs.company.com" -
+```
+
+lakeFS Enterprise Configuration:
+```yaml
+lakefsConfig: |
+  # Important: make sure to include the rest of your lakeFS Configuration here!
+
+  auth:
+    cookie_auth_verification:
+      auth_source: saml
+      friendly_name_claim_name: displayName
+      external_user_id_claim_name: samName
+      default_initial_groups:
+        - "Developers"
+    logout_redirect_url: "https://lakefs.company.com/logout-saml"
+    encrypt:
+      secret_key: shared-secrey-key
+    ui_config:
+      login_url: "https://lakefs.company.com/sso/login-saml"
+      logout_url: "https://lakefs.company.com/sso/logout-saml"
+      login_cookie_names:
+        - internal_auth_session
+        - saml_auth_session
+      rbac: external
+```
+
+Fluffy Configuration:
+```yaml
+fluffyConfig: &fluffyConfig |
+  logging:
+    format: "json"
+    level: "INFO"
+    audit_log_level: "INFO"
+    output: "="
+  installation:
+    fixed_id: fluffy-authenticator
+  auth:  
+    # better set from secret FLUFFY_AUTH_ENCRYPT_SECRET_KEY must be equal to what is used in lakeFS for auth_encrypt_secret_key
+    encrypt:
+      secret_key: shared-secrey-key    
+    logout_redirect_url: https://lakefs.company.com
+    post_login_redirect_url: https://lakefs.company.com
+    saml:
+      enabled: true 
+      sp_root_url: https://lakefs.company.com
+      sp_x509_key_path: '/etc/saml_certs/rsa_saml_private.cert'
+      sp_x509_cert_path: '/etc/saml_certs/rsa_saml_public.pem'
+      sp_sign_request: true
+      sp_signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
+      idp_metadata_url: "https://adfs-auth.company.com/federationmetadata/2007-06/federationmetadata.xml"
+      # idp_authn_name_id_format: "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
+      external_user_id_claim_name: samName
+      # idp_metadata_file_path: 
+      # idp_skip_verify_tls_cert: true
+```
+
+### OpenID Connect
```

Comment:
I wonder if these sections would be better presented as tabs (like we use in the current quickstart)
![CleanShot 2023-03-29 at 21 53 55](https://user-images.githubusercontent.com/3671582/228664853-e06a9b31-b2f3-4202-9a09-6a6367443fc3.png)

The page gets awfully long to scroll down otherwise.